build-farm/platform-specific-configurations/linux.sh

```diff
@@ -61,7 +61,10 @@ fi
 
 if [ "${ARCHITECTURE}" == "arm" ]
 then
-  export CONFIGURE_ARGS_FOR_ANY_PLATFORM="--with-jobs=4 --with-memory-size=2000 --disable-warnings-as-errors"
+  export CONFIGURE_ARGS_FOR_ANY_PLATFORM="--with-jobs=4 --with-memory-size=2000"
+  if [ "$JAVA_FEATURE_VERSION" -ge 11 ]; then
```

Comment:
Should that be `gt`? We built 11 successfully.

Reply:
nope it's `ge` essentially that flag is not supported in jdk8